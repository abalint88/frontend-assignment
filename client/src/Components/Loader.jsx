function Loader() {
    return (
        <div id="loader" className="loader">
            <div></div>
            <div></div>
            <div></div>
        </div>
    );
}

export default Loader;
