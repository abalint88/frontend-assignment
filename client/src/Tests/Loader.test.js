import "@testing-library/jest-dom/extend-expect";
import React from "react";
import renderer from 'react-test-renderer';

import Loader from "../Components/Loader";

test("Loader - should be rendered", () => {
    const component = renderer.create(
        <Loader />,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
