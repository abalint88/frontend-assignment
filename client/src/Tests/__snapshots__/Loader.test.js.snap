// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader - should be rendered 1`] = `
<div
  className="loader"
  id="loader"
>
  <div />
  <div />
  <div />
</div>
`;
